sdk/apimanagement/api-management-custom-widgets-tools/src/node/CustomWidgetBlobService.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlobServiceClient, BlockBlobUploadResponse } from "@azure/storage-blob";
+import { buildBlobConfigSrc, buildBlobDataSrc } from "../paths";
+import mime from "mime";
+
+type TConfig = Record<string, unknown>;
+
+/**
+ * A service wrapping ContainerClient class to simplify blob handling
+ */
+class CustomWidgetBlobService {
+  containerClient;
+  name;
+  pathWidget;
+  pathConfig;
```

Comment:
Is it worth adding types to these properties?

Reply:
type is inferred from usage in the constructor, I don't see benefit in specifying them here

Reply:
The benefit is in the documentation and readability, while it is easy for the language server to infer the types for a human it is not as quick.